zone/client.cpp

```diff
@@ -11997,3 +11997,18 @@ std::string Client::GetGuildPublicNote()
 
 	return gci.public_note;
 }
+
+void Client::MaxSkills()
+{
+	for (const auto &s : EQ::skills::GetSkillTypeMap()) {
+		auto current_skill_value = (
+			EQ::skills::IsSpecializedSkill(s.first) ?
+			50 :
```

Comment:
Is 50 hard-coded or defined somewhere?

Reply:
It’s the max a specialized skill can go before you set a specific specialization.

Reply:
Ah, yeah. May be worth adding to constants - no big deal.